hack/xref-helpmsgs-manpages

```diff
@@ -460,12 +463,11 @@ sub podman_help {
 sub podman_man {
     my $command = shift;
     my $subpath = "$Markdown_Path/$command.1.md";
-    my $manpath = "$FindBin::Bin/../$subpath";
```

Comment:
Pointless variable. The `cd` near top guarantees us being in the right dir to use relative paths.